### PR TITLE
Added new features and some minor changes

### PR DESCRIPTION
**Pull request by karan12990**
@mfilippov 

Sorry for the delay.

I am creating new PR and closing [PR](https://github.com/mfilippov/vimeo-dot-net/pull/160).

In this PR below changes are included.

- Allow user to fetch video by folder
- allow user to delete thumbnail of video
- Return picture base_link from pictures class
- Fixed property key of picture class link to base_link

Also, i have reverted licence and README file. Please take a look 

Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
-vimeo-dot-net
+vimeo-aspdot-net
```
Comment (mfilippov) [inline README.md:1]: Could you roll back this change.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
 =============
 
+## Fork of https://github.com/mfilippov/vimeo-dot-net.git
```
Comment (mfilippov) [inline README.md:4]: And fork info.

**Review by mfilippov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (karan12990) [reply README.md]: > Could you roll back this change.

done. please check now.
thanks.

File: README.md (hunk 2 above)
Comment (karan12990) [reply README.md:4]: > And fork info.

you means do i need to delete this line ? As i can't see this line in your file. 

Do let me know if i needs to delete this line from my PR ?


image

Comment (mfilippov) [reply README.md:4]: Yes, please delete this chunk.  It looks strange in this repo.

**Review by mfilippov — APPROVED** [review]
LGTM. I will check what is wrong with tests on this week.

**Comment by karan12990** [comment]
@mfilippov 

Did you get a chance to look into this ?

**Comment by mfilippov** [comment]
I checked tests locally all work ok. Looks like a CI issue.

**Comment by mfilippov** [comment]
Thank you for your contribution!

**Comment by mfilippov** [comment]
@karan12990 Your PR available in v2.1.2 on nuget.org

**Comment by karan12990** [comment]
@mfilippov 

Thank you.

**Pull request merged**